### PR TITLE
[DOCS - ekapner] Add LIKE, swap out gifs for inclusive lang

### PR DESCRIPTION
**Pull request by ekapner**
### What does this PR do?

Add documentation for LIKE, swap out gifs to abide by inclusive language initiative.

### What issues does this PR fix or reference?
@W-8908587@

### CODE REVIEW

File: docs/_articles/en/soql/soql-builder.md
Comment (jonnyhork) [inline docs/_articles/en/soql/soql-builder.md:18]: Is this supposed to be a new line or new bullet? It renders on the same line as 
`LIMIT clause`.

File: docs/_articles/en/soql/soql-builder.md
Comment (jgellin-sf) [inline docs/_articles/en/soql/soql-builder.md:90]: Nice balance here between linking the LIKE variations and too much explanation.

File: docs/_articles/en/soql/soql-builder.md
Comment (jonnyhork) [inline docs/_articles/en/soql/soql-builder.md:53]: The `the Switch Between SOQL Builder and Text Editor.` seems weird to me without referencing _what_ it is, such as an icon or button. If They can't see the gif then the reader will have no idea what to look for.

**Review by jonnyhork — COMMENTED** [review]
Looks good, except one formatting thing and possible clarification.

File: docs/_articles/en/soql/soql-builder.md
Comment (jshackell-sfdc) [inline docs/_articles/en/soql/soql-builder.md:18]: Small suggested edit (to shorten sentence a bit):  "To dig deeper into SOQL syntax or to build more complex queries in the text editor, see the [SOQL and SOSL Reference guide](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_sosl_intro.htm)."

File: docs/_articles/en/soql/soql-builder.md
Comment (jshackell-sfdc) [inline docs/_articles/en/soql/soql-builder.md:53]: Edit: remove "the".

File: docs/_articles/en/soql/soql-builder.md
Comment (jshackell-sfdc) [inline docs/_articles/en/soql/soql-builder.md:76]: tiny suggestion, because this is a complex sentence.  "... **by using the LIKE operator with wildcards** to match partial text strings."  (bold is mine to show what I changed.).

File: docs/_articles/en/soql/soql-builder.md
Comment (jshackell-sfdc) [reply docs/_articles/en/soql/soql-builder.md:90]: +1

File: docs/_articles/en/soql/soql-builder.md
Comment (ekapner) [reply docs/_articles/en/soql/soql-builder.md:18]: Thanks, @jonnyhork, added a line break - will validate on staging.

File: docs/_articles/en/soql/soql-builder.md
Comment (ekapner) [reply docs/_articles/en/soql/soql-builder.md:53]: Called it an icon.

File: docs/_articles/en/soql/soql-builder.md
Comment (ekapner) [reply docs/_articles/en/soql/soql-builder.md:76]: Appleby, Appleton, heh heh heh - went ahead and changed it. 

@jshackell-sfdc - great edits, thanks.

**Comment by ekapner** [comment]
Hey @jshackell-sfdc - made your edits - please approve at your earliest convenience - need a doc maintainer approval to merge.

**Review by jshackell-sfdc — APPROVED** [review]

**Pull request merged**
